docs/docs/guide/usingFreighterBrowser.md

```diff
@@ -21,191 +21,12 @@ _NOTE:_ You must use version `1.1.2` or above
 
 This will expose a global variable called `window.freighterApi` that will contain our library.
 
-Now let's dig into what functionality is available to you:
+The call signatures will be exactly the same as the [node version](./usingFreighterNode), but you will call the methods directly from `window.freighterApi`:
 
-### isConnected
```

Comment:
lol I don't know what I was thinking duping these docs in both `usingFreighterBrowser` and `usingFreighterNode`. They were getting out of sync (obviously). 

Devs should just be able to use the node examples to figure it out